AngularApp/src/app/diagnostic-data/components/markdown/markdown.component.html

```diff
@@ -1,10 +1,10 @@
 <ng-template #header>
     {{renderingProperties.title}}
-    <button class="btn btn-default" style="float:right;margin-top: -7px" *ngIf="!isPublic" (click)="copyMarkdown()">
+    <button class="btn btn-default" style="float:right;margin-top: -7px" *ngIf="!isPublic" (click)="logCopyMarkdown();copyMarkdown()">
```

Comment:
same here, can you just call logCopyMarkdown() from inside copyMarkdown()? same for logOpenEmail?